ansible/roles/products/common/demo-icons/tasks/main.yml

```diff
@@ -72,8 +72,6 @@
   - name: Make Close Right Hand desktop icon Trusted
     shell: gio set "{{ demo_icon_folder }}/Close Right Hand.desktop" "metadata::trusted" yes
     when:
-      - ansible_distribution|string == 'Ubuntu'
-      - ansible_distribution_release|string == 'bionic'
```

Comment:
why are these being removed?

Reply:
Since you are removing support for lower versions, are you making a check in the script to prevent this from running before initiating the ansible roles?